test/Microsoft.AspNetCore.Razor.Evolution.Test/TestFiles/IntegrationTests/BasicIntegrationTest/HelloWorld.ir.txt

```diff
@@ -1,5 +1,8 @@
 Document - 
+    Checksum - 
     NamespaceDeclaration -  - 
+        UsingStatement -  - System
+        UsingStatement -  - System.Threading.Tasks
         ClassDeclaration -  -  -  -  - 
             RazorMethodDeclaration -  -  -  -  - 
-                HtmlContent - (0:0,0 [13] ) - Hello, World!
+                HtmlContent - (0:0,0 [13] TestFiles/IntegrationTests/BasicIntegrationTest/HelloWorld.cshtml) - Hello, World!
```

Comment:
I'm kinda wondering if we should 'alias' this for readability when it's based on the 'primary' file. Or maybe just truncate it down to the last segment.